basket/src/components/Profile.tsx

```diff
@@ -0,0 +1,28 @@
+type Session = {
+  loginUser: {
+    id: number;
+    name: string;
+  };
+  cart: Array<{ id: number; name: string; price: number }>;
+};
+
+type MyProps = {
+  session: Session;
+  logout: () => void;
+};
+
+export const Profile = ({ session, logout }: MyProps) => {
+  return (
+    <div>
+      <p>Logged in as: {session.loginUser.name} ({session.loginUser.id})</p>
+      <ul>
+        {session.cart.map((ele) => (
+          <li style={{listStyle: 'none'}} key={ele.id}>
+            <strong>{ele.name}</strong>
+            <span>{ele.price}</span>
+          </li>))}
```

Comment:
1. map에서 return문 없이 깔끔하게 작성해준 것 👍 
2. JSX는 가능한 최대한 가독성을 높이기!
```suggestion
        {session.cart.map(({ id, name, price }) => (
          <li style={{listStyle: 'none'}} key={id}>
            <strong>{name}</strong>
            <span>{price}</span>
          </li>))}
```